packages/xo-web/src/common/select-boot-firmware.js

```diff
@@ -0,0 +1,48 @@
+import _ from 'intl'
+import PropTypes from 'prop-types'
+import React from 'react'
+import { confirm } from 'modal'
+import { injectState, provideState } from 'reaclette'
+import { noop } from 'utils'
+import { VM_BOOT_FIRMWARES } from 'xo'
+
+const withState = provideState({
+  effects: {
+    handleBootFirmwareChange(
+      __,
+      {
+        target: { value },
+      }
+    ) {
+      if (value !== '') {
```

Comment:
What if the default value is not selected and the user want to use it?

Reply:
I don't understand the logic of your question :exclamation: 
default value not selected :arrow_right: user want to use it :exclamation:

Reply:
You're right :heavy_check_mark: I fixed it